projects/pedal_board/src/brake_calib.c

```diff
@@ -0,0 +1,56 @@
+#include "brake_calib.h"
+#include "ads1015.h"
+#include "brake_data.h"
+#include "log.h"
+#include "pedal_calib.h"
+#include "pedal_data.h"
+#include "pedal_events.h"
+#include "string.h"
+#include "wait.h"
+
+int32_t average_value;
+static Ads1015Storage *s_ads1015_storage;
+
+static void prv_callback_channel(Ads1015Channel ads1015, void *context) {
+  BrakeCalibrationStorage *storage = context;
+  int16_t reading = 0;
+  ads1015_read_raw(get_ads1015_storage(), get_pedal_data_storage()->brake_channel, &reading);
```

Comment:
#define brake channel in `pedal_data` and use it directly here. Get rid of `get_pedal_data_storage`.